pkg/daemon/ceph/mon/config.go

```diff
@@ -1,104 +1,30 @@
-/*
```

Comment:
licensing?

Reply:
I removed the license because `godoc` outputs the license, sometimes multiple times, for files like ...
```
blaine@grim:~/development/rasputin/rook> godoc github.com/rook/rook/cmd/rook
COMMAND DOCUMENTATION

    Copyright 2018 The Rook Authors. All rights reserved.

    Licensed under the Apache License, Version 2.0 (the "License"); you may
    not use this file except in compliance with the License. You may obtain
    a copy of the License at

        http://www.apache.org/licenses/LICENSE-2.0

    Unless required by applicable law or agreed to in writing, software
    distributed under the License is distributed on an "AS IS" BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.

    Copyright 2016 The Rook Authors. All rights reserved.

    Licensed under the Apache License, Version 2.0 (the "License"); you may
    not use this file except in compliance with the License. You may obtain
    a copy of the License at

        http://www.apache.org/licenses/LICENSE-2.0

    Unless required by applicable law or agreed to in writing, software
    distributed under the License is distributed on an "AS IS" BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.

SUBDIRECTORIES

        ceph
        cockroachdb
        minio
        nfs
        rook
        version

```

I'm not sure if there is a way to add godoc to the package and still retain the license text.

Reply:
Interesting, I wonder what other projects do with godoc since we still need the licensing in all the source files.

Reply:
It turns out the solution is easy to keep both the license and have package documentation. `godoc` only sees the comment which "touches" the `package ...` definition as the package comment so the below works as I would expect, not outputting the copyright but still outputting the package doc:

```go
/* Copyright, Lazy Industries, Ltd. Inc. LLC. */

/* Package himom says hi to your mother every day via SMS */
package himom
```